test/functional/wallet_listreceivedby.py

```diff
@@ -26,48 +26,52 @@ def skip_test_if_missing_module(self):
         self.skip_if_no_cli()
 
     def run_test(self):
+        self.test_listreceivedbyaddress()
+        self.test_listreceivedby()
+        self.test_getreceivedby()
+        self.test_receivedbylabel()
+        self.test_coinbase_inclusion()
+
+    def test_listreceivedbyaddress(self):
         # save the number of coinbase reward addresses so far
         num_cb_reward_addresses = len(self.nodes[1].listreceivedbyaddress(minconf=0, include_empty=True))
 
-        self.log.info("listreceivedbyaddress Test")
+        self.log.info("Tests listreceivedbyaddress")
+        self.log.info("- respects minimum confirmations")
 
         # Send from node 0 to 1
         addr = self.nodes[1].getnewaddress()
         txid = self.nodes[0].sendtoaddress(addr, 0.1)
         self.sync_all()
 
         # Check not listed in listreceivedbyaddress because has 0 confirmations
-        assert_array_result(self.nodes[1].listreceivedbyaddress(),
-                            {"address": addr},
-                            {},
-                            True)
+        assert_array_result(self.nodes[1].listreceivedbyaddress(), {"address": addr}, {}, True)
         # Bury Tx under 10 block so it will be returned by listreceivedbyaddress
         self.generate(self.nodes[1], 10)
-        assert_array_result(self.nodes[1].listreceivedbyaddress(),
-                            {"address": addr},
-                            {"address": addr, "label": "", "amount": Decimal("0.1"), "confirmations": 10, "txids": [txid, ]})
+        expected = {"address": addr, "label": "", "amount": Decimal("0.1"), "confirmations": 10, "txids": [txid, ]}
+        assert_array_result(self.nodes[1].listreceivedbyaddress(), {"address": addr}, expected)
         # With min confidence < 10
-        assert_array_result(self.nodes[1].listreceivedbyaddress(5),
-                            {"address": addr},
-                            {"address": addr, "label": "", "amount": Decimal("0.1"), "confirmations": 10, "txids": [txid, ]})
+        assert_array_result(self.nodes[1].listreceivedbyaddress(5), {"address": addr}, expected)
         # With min confidence > 10, should not find Tx
         assert_array_result(self.nodes[1].listreceivedbyaddress(11), {"address": addr}, {}, True)
 
         # Empty Tx
+        self.log.info("- includes empty addresses")
         empty_addr = self.nodes[1].getnewaddress()
-        assert_array_result(self.nodes[1].listreceivedbyaddress(0, True),
-                            {"address": empty_addr},
-                            {"address": empty_addr, "label": "", "amount": 0, "confirmations": 0, "txids": []})
+        expected_tx = {"address": empty_addr, "label": "", "amount": 0, "confirmations": 0, "txids": []}
+        assert_array_result(self.nodes[1].listreceivedbyaddress(0, True), {"address": empty_addr}, expected_tx)
 
-        # No returned addy should be a change addr
+        self.log.info("- does not return changes addresses")
```

Comment:
“changes addresses” -> “change addresses” [“change addresses” is the correct term for output addresses, “changes” is a verb here and incorrect]